Gemfile

```diff
@@ -1,9 +1,11 @@
 source 'https://rubygems.org'
 
+
 ruby '2.1.5'
 # Bundle edge Rails instead: gem 'rails', github: 'rails/rails'
 gem 'rails', '4.2.0'
 # Use mysql as the database for Active Record
```

Comment:
sure will do that.

On Mon, Mar 16, 2015 at 12:32 PM, DEVARAJ K <notifications@github.com>
wrote:

> In Gemfile
> <https://github.com/sanchaya/sanchaya-api/pull/2#discussion_r26462932>:
>
> > @@ -1,9 +1,11 @@
> >  source 'https://rubygems.org'
> >
> > +
> >  ruby '2.1.5'
> >  # Bundle edge Rails instead: gem 'rails', github: 'rails/rails'
> >  gem 'rails', '4.2.0'
> >  # Use mysql as the database for Active Record
>
> Can you take out pagination for now ?
> We will use pagination in web. lets think of api later
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/sanchaya/sanchaya-api/pull/2/files#r26462932>.
>